nistats/tests/test_utils.py

```diff
@@ -133,6 +133,12 @@ def create_fake_bids_dataset(base_dir='', n_sub=10, n_ses=2,
                              tasks=['localizer', 'main'],
                              n_runs=[1, 3], with_derivatives=True,
                              with_confounds=True):
+    """Returns a fake bids dataset directory with dummy files
+
+    In the case derivatives are included, they come with two spaces and
+    variants. Spaces are 'MNI' and 'T1w'. Variants are 'some' and 'other'.
```

Comment:
you're probably referring to BIDS specs here ?

Reply:
not really, they are dummy specs